frontend/apps/web/components/site-header/AccountStatusHandler.tsx

```diff
@@ -35,3 +45,69 @@ export function AccountStatusHandler(props: Props) {
     </div>
   );
 }
+
+interface TrialCountdownProps {
+  createdDate: number;
+  isAccountStatusValidResp: IsAccountStatusValidResponse | undefined;
+}
+
+const TRIAL_DURATION_DAYS = 14;
```

Comment:
Ideally we could do most or all of this on the server so we don't have this copied in two places.